### PR TITLE
Add skip baremetalds e2e tests environment variable

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/assisted/baremetalds-assisted-workflow.yaml [hunk 1]
```diff
@@ -6,7 +6,11 @@ workflow:
       - chain: baremetalds-assisted-setup
     test:
       - ref: baremetalds-e2e-conf
-      - ref: baremetalds-e2e-test
+      - env:
+        - name: SKIP_E2E_TEST
+          default: "true"
+          documentation: Wheter to skip the e2e tests
```
Comment (osherdp) [inline ci-operator/step-registry/baremetalds/assisted/baremetalds-assisted-workflow.yaml:12]: typo

Comment (osherdp) [reply ci-operator/step-registry/baremetalds/assisted/baremetalds-assisted-workflow.yaml:12]: also, this sentence is not complete

**Review by osherdp — CHANGES_REQUESTED** [review]

**Review by osherdp — APPROVED** [review]
/lgtm

**Comment by YuviGold** [comment]
Resolved breaking serial conformance tests that were introduced on https://github.com/openshift/release/pull/16075 by splitting the conf step on https://github.com/openshift/release/pull/16207

**Pull request closed without merging**